controllers/fenceagentsremediation_controller.go

```diff
@@ -225,6 +239,7 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		!meta.IsStatusConditionTrue(far.Status.Conditions, commonConditions.SucceededType) {
 		// Fence agent action succeeded, and now we try to remove workloads (pods and their volume attachments)
 		r.Log.Info("Manual workload deletion", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
+		commonEvents.NormalEvent(r.Recorder, node, v1alpha1.EventReasonDeleteResources, v1alpha1.EventMessageDeleteResources)
```

Comment:
Similar to the note about the message "Fence agent executed" above. This event is emitted before trying to delete the resources, it should reflect that, or it can be moved after the deletion request succeeded.

Reply:
The idea was to send an event every time we tried to delete resources in case it didn't succeed.

Reply:
What makes you think it is going to be a warning event? It uses `commonEvents.NormalEvent`.

Reply:
I meant, this message originally made the reader think the deletion was already successful, while it had to be done yet.
The new message reflects instead that we will attempt to delete the resources :+1: